### PR TITLE
Implement landing page in frontend

### PR DESCRIPTION
**Pull request by jotjern**

### CODE REVIEW

File: apps/web/src/components/organisms/EventsPane/MainEventPane.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { type Event } from "@dotkomonline/types"
+
+function formatDate(date: Date): string {
+  const day = date.getDate()
+  const months = ["jan", "feb", "mar", "apr", "may", "jun", "jul", "aug", "sep", "oct", "nov", "dec"]
+  const month = months[date.getMonth()]
+  const year = date.getFullYear()
+
+  return `${day}. ${month} ${year}`
+}
```
Comment (junlarsen) [inline apps/web/src/components/organisms/EventsPane/MainEventPane.tsx:10]: Would recommend using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

File: apps/web/src/components/organisms/EventsPane/index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { EventsPane } from "./EventsPane"
+
+export { EventsPane }
```
Comment (junlarsen) [inline apps/web/src/components/organisms/EventsPane/index.tsx:3]:
```suggestion
export { EventsPane } from "./EventsPane"
```

File: packages/core/src/modules/event/event-repository.ts [hunk 3]
```diff
@@ -40,6 +41,17 @@ export class EventRepositoryImpl implements EventRepository {
...
+  async getAllOrderedByDate(ascending: boolean, take: number) {
+    const query = this.db
+      .selectFrom("event")
+      .selectAll()
+      .orderBy("start", ascending ? "asc" : "desc")
+      .limit(take)
+
+    const events = await query.execute()
+    return events.map((e) => mapToEvent(e))
+  }
```
Comment (junlarsen) [inline packages/core/src/modules/event/event-repository.ts:53]: Should take a cursor object here instead and use orderedQuery

File: packages/gateway-trpc/src/modules/event/event-router.ts [hunk 4]
```diff
@@ -47,6 +47,24 @@ export const eventRouter = t.router({
       return event
     }),
 
+  allOrderedByDate: publicProcedure
+    .input(
+      z
+        .object({ take: z.number().default(10), ascending: z.boolean().default(false) })
+        .default({ take: 10, ascending: false })
+    )
```
Comment (junlarsen) [inline packages/gateway-trpc/src/modules/event/event-router.ts:55]: Replace with Cursor in repository and use PaginateInputSchema here

**Review by junlarsen — COMMENTED** [review]
Nice work. I left some code comments, let me know on Slack or something if you want guidance with the cursors

**Comment by henrikskog** [comment]
DOT-314

**Comment by henrikskog** [comment]
https://linear.app/dotkom/issue/DOT-314/create-front-page

**Pull request closed without merging**